README.md

```diff
@@ -8,3 +8,165 @@ You can choose to use either the:
+   ```
+2. Add a polyfill for `global` by adding the following to `index.html`.
+   ```html
+   <script>
+     if (global === undefined) {
+       var global = window;
+     }
+     if (module === undefined) {
+       var module = {};
+     }
+   </script>
+   ```
+3. Set `base` to `./` in `vite.config.ts` so the content can be hosted in a subfolder:
+   e.g.
+
+   ```ts
+   import { defineConfig } from "vite";
+
+   export default defineConfig({
+     base: "./",
+     ...
+   });
+   ```
+
+4. If using Typescript, add the following options to `tsconfig.js` to support importing `cogs-plugin-manifest.js`:
+   ```json
+   {
+     "compilerOptions": {
+       "checkJs": true
+     }
+   }
+   ```
+5. Add a `cogs-plugin-manifest.js` file to the `src` folder as described above
+6. Symlink `public/cogs-plugin-manifest.js` to `src/cogs-plugin-manifest.json` so that it is included in the build output:
+   ```s
+   # macOS / Linux
+   ln -s ../src/cogs-plugin-manifest.js public/cogs-plugin-manifest.js
+   # Windows
+   mklink /H public\cogs-plugin-manifest.js src\cogs-plugin-manifest.js
+   ```
+7. If you are using React, remove `<React.StrictMode>` from your top-level component to avoid issues with multiple websockets during development.
+8. Add the [COGS SDK](https://github.com/clockwork-dog/cogs-sdk/) to your project. (See [#quick-start](Quick start).)
+
+When importing your manifest file in your source code be sure to use a wildcard import as follows:
+
+```js
+import * as manifest from './cogs-plugin-manifest.js';
+```
+
+### Create React App template (deprecated)
+
+See https://github.com/clockwork-dog/cra-template-cogs-client
+
+### Create React App (deprecated)
+
+1. Create a project with `create-react-app` as usual
+   e.g.
+   ```s
+   yarn create react-app my-react-plugin --template typescript
+   ```
+2. Set `homepage` to `./` in `package.json` so the content can be hosted at in a subfolder:
+   e.g.
+   ```json
+   {
+     "homepage": "./",
+     ...
+   }
+   ```
+3. Add a `cogs-plugin-manifest.js` file to the `src` folder as described above
+4. Symlink `public/cogs-plugin-manifest.js` to `src/cogs-plugin-manifest.json` so that it is included in the build output:
+   ```s
+   # macOS / Linux
+   ln -s ../src/cogs-plugin-manifest.json public/cogs-plugin-manifest.js
+   # Windows
+   mklink /H public\cogs-plugin-manifest.js src\cogs-plugin-manifest.js
+   ```
+5. If you are using React, remove `<React.StrictMode>` from your top-level component to avoid issues with multiple websockets during development.
+6. Add the [COGS SDK](https://github.com/clockwork-dog/cogs-sdk/) to your project. (See [#quick-start](Quick start).)
+
+## Tips
+
+We recommend that you do not rely on loading content over the internet for maximum reliability. This means your content will load correctly and quickly even when internet connectivity at your venue is not perfect. You should include all CSS, fonts, Javascript, etc in your plugin.
```

Comment:
I would include a title for this as a summary and to easily allow adding more tips in the future.

```suggestion
### Don't load assets over the internet

We recommend that you do not rely on loading content over the internet for maximum reliability. This means your content will load correctly and quickly even when internet connectivity at your venue is not perfect. You should include all CSS, fonts, Javascript, etc in your plugin.
```